en/doc_directory.md

```diff
@@ -0,0 +1,38 @@
+# Document Directory
+
+SCRF's Document Directory is a comprehensive guide of all SCRF documentation.
+
+## Awards, Bounties, Grants
+
+* [Research Summary Grants](en/grants_general_contributors.md)
```

Comment:
If this file is _in_ the `en` directory then the prefix on the URL needs to be removed.

Reply:
i.e. 

bad: `[Research Summary Grants](en/grants_general_contributors.md)`
good: `[Research Summary Grants](grants_general_contributors.md)`

Reply:
Fixed